server/jetstream_helpers_test.go

```diff
@@ -1194,13 +1194,13 @@ func (c *cluster) waitOnServerHealthz(s *Server) {
 	c.t.Helper()
 	expires := time.Now().Add(30 * time.Second)
 	for time.Now().Before(expires) {
-		hs := s.healthz()
+		hs := s.healthz(&HealthzOptions{})
```

Comment:
Would pass `nil` here since you support options being nil (and create them empty in the healthz() function itself)

Reply:
updated